### ISSUE TITLE
Kika keyboard is suspect

### ISSUE THREAD

**Issue by jaykru** [phhusson/unihertz_titan#2]
Kika seems to have some privacy concerns https://www.buzzfeednews.com/article/craigsilverman/android-apps-cheetah-mobile-kika-kochava-ad-fraud. I also have no clue what the license is, but I presume it's not FOSS. Has anyone looked into alternatives?  AnySoftKeyboard supposedly has soft keyboard support, but I haven't managed to enable functionality similar to what Kika offers on the Titan.

**Comment by phhusson** [comment]
I agree it is suspect. (well i pretty much never trust any oem app, even less third party)
My take on this is that it does a pretty great job, my main worry is whether it is leaking data to the internet, which can be solved easily by removing it its internet connection permission. (most custom rom have a option for it, though not my aosp gsi which would require repacking the apk).
Though there might be another issue that it could be leaking data to other apps.

Having an opensource alternative would be very nice, but it feels like it is quite some job, and I don't see improvements we could bring by being opensource.

Jul 5, 2020 8:51:20 AM Jay Kruer <notifications@github.com>:
